run/pcap2john.py

```diff
@@ -807,6 +807,69 @@ def pcap_parser_glbp(fname):
     f.close()
 
 
+# Parts are borrowed from "module_tacacs_plus.py" from Loki project which is
+# Copyright 2015 Daniel Mende <dmende@ernw.de>. See tacacs_plus_fmt.plug.c for
+# licensing information.
```

Comment:
Oh, that. I think we should clearly add Daniel's 3-clause BSD license right into this place in the Python script then. Instead of putting it into the JtR format file, which he didn't write.

Reply:
Sure. I will add the license blurb into the Python script shortly.